aws-datastore/src/androidTest/java/com/amplifyframework/datastore/storage/SynchronousStorageAdapter.java

```diff
@@ -91,10 +98,22 @@ public static SynchronousStorageAdapter create(
      */
     @SuppressWarnings("UnusedReturnValue")
     public List<ModelSchema> initialize(@NonNull Context context) throws DataStoreException {
+        Log.d("initialize", "inside initialize");
```

Comment:
Try to use `Amplifyer.Logging` in the tests, there are existing patterns in the codebase. The tag here should end up looking like `aws-amplify:datastore:test`, if memory serves. `"initialize"` would be a curious tag.

Reply:
Left over from debugging removed. Thanks for catching it.